divisiondetector/datasets.py

```diff
@@ -1,42 +1,81 @@
-from torch.utils.data.dataset import Dataset
-import numpy as np
+import os
+import pandas as pd
+from torch.utils.data import Dataset
+from utils.gp_pipeline import GPPipeline
 
 class DivisionDataset(Dataset):
+    def __init__(self, label_path, img_path, window_size=(100,100,100), time_window=(1, 1), mode='ball', ball_radius=(10, 10, 10)):
+        def __getlabels(label_path, div_path):
+            '''
+            Load and process label data
+            '''
+            raw_df = pd.read_csv(label_path, encoding='unicode_escape')
 
-    def __init__(self):
+            print("Data loaded.")
```

Comment:
Let's use logging instead of prints. 

outside of the function
`import logging`
`logger = logging.getLogger()`

inside, where you want to print
`logger.info('Data loaded')`